src/plugin.js

```diff
@@ -1,6 +1,7 @@
 import { resolve, dirname, relative } from 'path';
 import { ResolverFactory, SyncNodeJsInputFileSystem } from 'enhanced-resolve';
 import { parse } from 'babylon';
+import template from 'lodash.template';
```

Comment:
In this form we get advantage right now without tree shaking, with tree shaking as I understand we could just write
`import { template } from 'lodash'` and just `template` module will be imported. (But I haven't use tree shaking so may mistake)